packages/desktop-client/src/components/common/Modal.tsx

```diff
@@ -489,3 +492,12 @@ export function ModalCloseButton({ onPress, style }: ModalCloseButtonProps) {
     </Button>
   );
 }
+
+function ErrorFallback() {
```

Comment:
It would be good to make use of the `error` and `resetErrorBoundary` props ([here](https://arc.net/l/quote/htlzilbu)) to provide a better experience. Specifically, check out the [`FatalError`](https://github.com/actualbudget/actual/blob/master/packages/desktop-client/src/components/FatalError.tsx#L194) component for how the top-level error boundary works